src/types/index.js

```diff
@@ -75,6 +75,29 @@ type StackLocal = {
   value: any
 };
 
+interface Number<T> {
```

Comment:
I noticed that this doesn't work in Flowtype, `Number` refers to the JavaScript Number object. I will rename it `NumberInterface`.